packages/payment-processor/test/payment/encoder-payment.test.ts

```diff
@@ -446,3 +502,90 @@ describe('Payment encoder handles ERC777 Stream', () => {
     });
   });
 });
+
+describe('Payment encoder handles Meta PN', () => {
+  describe('Error cases', () => {
+    it('Should not be possible to encode a transaction when passing an invalid pn identifier', () => {
+      expect(() =>
+        encodeRequestPayment(validMetaRequest, provider, {
+          conversion: alphaConversionSettings,
+          pnIdentifier: 'unknown',
```

Comment:
This test makes it look like it throws because "unknown" is an invalid pnId, but it would also through with "eth-input" for instance, which is valid but missing.

Reply:
`eth-input` is not valid the `pnIdentifier` within the `meta-pn` is a unique identifier to identify each sub-pn, it's not the payment network identifiers like `any-to-erc20` `eth-input` ...